ntoskrnl/ke/amd64/cpu.c

```diff
@@ -350,6 +353,7 @@ KiGetFeatureBits(VOID)
 VOID
 KiReportCpuFeatures(IN PKPRCB Prcb)
 {
+    ULONG64 FeatureBits = Prcb->FeatureBits | ((ULONG64)Prcb->FeatureBitsHigh << 32);
```

Comment:
This fix could be committed separately/first, as it is already needed for `KF_SSE4_2` for example.